### ISSUE TITLE
visualization results on the cifar10 dataset are very poor

### ISSUE THREAD

**Issue by Hanawh** [InterDigitalInc/CompressAI#24]
Thanks for your great work! I trained a simple Auto Encoder with cifar10, and my loss dropped normally during training. After training, I updated the parameters of the entropy model. When I want to visualize my results, if there is no model.eval(), the reconstructed picture is normal, but if model.eval() is used, it will be very bad. Looking forward to your reply.

**Comment by jbegaint** [comment]
hi, thanks for the report. Can you share the hyper-parameters and the training schedule you used?

**Comment by Hanawh** [comment]
I use the sample/train.py to train cifar10，
optimizer = optim.Adam(net.parameters(), lr=1e-4)
aux_optimizer = optim.Adam(net.aux_parameters(), lr=1e-3)

I also added ReduceLROnPlateau according to  "Models were trained with a batch size of 16 or 32, and an initial learning rate of 1e-4 for approximately 1-2M steps. The learning rate of the main optimizer is then divided by 2 when the evaluation loss reaches a plateau (we use a patience of 20 epochs). This can be implemented by using PyTorch ReduceLROnPlateau learning rate scheduler."

when I use bmshj2018_factorized, the result on cifar10 is good which test loss(0.2) and train loss(0.2) are basically the same. 
But when I use AutoEncoder, there is a big gap between test loss(0.7) and train loss(0.2).

**Comment by Hanawh** [comment]
By the way，when I change the batchsize, does the learning rate need to be adjusted proportionally? Thank you so much for your reply and guidance.

**Comment by jbegaint** [comment]
thanks, i'll try to reproduce the issue.

regarding the batch-size, we tend to use the same LR with for a batch-size in (8, 16, 32).

**Comment by Hanawh** [comment]
Thanks for your reply!

**Comment by Hanawh** [comment]
Sorry to bother you again. I have another question. I want to know what the value of lambda in loss function depends on.

**Comment by jbegaint** [comment]
See https://github.com/InterDigitalInc/CompressAI/issues/27
